tests/testthat/test-outfiles.R

```diff
@@ -9,6 +9,7 @@ expect_messages_to_include <- function(haystack, needles) {
 test_that("expected outfiles are written and messaged, venue = 'gh'", {
   skip_on_cran()
   local_temp_wd()
+  local_reprex_quiet(FALSE)
 
   msg <- capture_messages(
```

Comment:
I still have a few places where I explicitly use `capture_messages()` which is superseded.

But I truly want to state some specific expectations about the message content and/or I need to dig a piece of information out of the message (e.g. a path). I'm not sure all this is really covered by the golden test functions.